src/validator.cpp

```diff
@@ -273,6 +273,38 @@ struct Validator::ValidatorImpl
      */
     bool isSupportedMathMLElement(const XmlNodePtr &node);
 
+    /**
+    * @brief Validate that equivalent variable pairs in the @p model
+    * have equivalent units.
+    * Any issues will be logged in the @c Validator.
+    *
+    * Any difference in base units is reported as an issue in the @c Validator, but the multiplier difference does not trigger a validator issue.
+    * Where the base units are equivalent, the multiplier may be interpreted as units_of_v1 = (10^multiplier)*units_of_v2
+    *
+    * @param model The model containing the variables
+    * @param v1 The variable which may contain units.
+    * @param v2 The equivalent variable which may contain units.
+    * @param hints String containing issue messages to be passed back to the calling function for logging.
+    * @param multiplier Double returning the effective multiplier mismatch between the units.
+    */
+    // KRM bool unitsAreEquivalent(const ModelPtr &model, const VariablePtr &v1, const VariablePtr &v2, std::string &hints, double &multiplier);
```

Comment:
What is this? Left over?

Reply:
See lines 38-52.

Reply:
Yep, whoops!  Done.  There's an issue already to have the validator use the functions for unit checking which are now in the Units class.  I won't change that in this one.